### PR TITLE
src: add missing /* glsl */ & string template

### PR DESCRIPTION
**Pull request by yushijinhun**
It's all in `src/` so there is no compatibility problem.

### CODE REVIEW

File: src/renderers/WebGLCubeRenderTarget.js [hunk 1]
```diff
@@ -45,46 +45,33 @@ WebGLCubeRenderTarget.prototype.fromEquirectangularTexture = function ( renderer
 			tEquirect: { value: null },
 		},
 
-		vertexShader: [
+		vertexShader: /* glsl */`
+varying vec3 vWorldDirection;
 
-			"varying vec3 vWorldDirection;",
```
Comment (Mugen87) [inline src/renderers/WebGLCubeRenderTarget.js:50]: The existing formatting in `WebGLCubeRenderTarget` is actually more readable than the new one. Probably better to align `PMREMGenerator` to `WebGLCubeRenderTarget`.

Comment (yushijinhun) [reply src/renderers/WebGLCubeRenderTarget.js:50]: Maybe we need a GLSL code formatter.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

File: src/renderers/WebGLCubeRenderTarget.js (hunk 1 above)
Comment (Mugen87) [reply src/renderers/WebGLCubeRenderTarget.js:50]: Please no over-engineering 😉

**Pull request merged**